css/subject-viewer.styl

```diff
@@ -107,6 +107,13 @@
           background-color: #bbc0c0
           color: #575959
 
+  &.subject-viewer--invert
+    // Talk
+    .subject-container
+      .subject-image-frame:only-child
+        img, video
```

Comment:
Do CSS filters work on video?

Reply:
It does! There's an issue with which class I'm applying the filter to, I'll fix.